### PR TITLE
Fixes bz#1187815 NICs with Subnets if assigned shown on Assigned Tab

### PR DESCRIPTION
**Pull request by knowncitizen**

### CODE REVIEW

**Comment by sseago** [comment]
I'm assuming that the correct presentation for a nic with no subnet is something like "eth0 ()" -- if so, then ACK.

File: app/models/staypuft/concerns/host_interface_management.rb [hunk 1]
```diff
@@ -17,6 +17,16 @@ def interfaces_identifiers
         interfaces
       end
 
+      def interfaces_identifiers_with_subnets
+        interfaces = [ [self.primary_interface, self.subnet.name] ]
+        if self.respond_to?(:interfaces)
+          interfaces += self.interfaces.where("type <> 'Nic::BMC'").physical.map do |interface|
+            [interface.identifier, ((sub = interface.subnet) ? sub.name :  "")]
+            end
```
Comment (bcrochet) [inline app/models/staypuft/concerns/host_interface_management.rb]: Indentation is off here...

**Comment by bcrochet** [comment]
Other than the indentation, LGTM.

**Comment by bcrochet** [comment]
Visual ACK

**Pull request merged**